services/portal/GWAS/GWASTasks.js

```diff
@@ -15,6 +16,42 @@ module.exports = {
         I.saveScreenshot('GWAS_page.png');
     },
 
+    goToAnalysisPage() {
+        I.amOnPage(GWASUIAppProps.analysisPath);
+        I.wait(5);
+        I.saveScreenshot('analysis.png');
+    },
+
+    async selectTeamProject1() {
+        I.seeElement(GWASUIAppProps.TeamProjectSelectorBox);
+        I.click(GWASUIAppProps.TeamProjectSelectorDropdown);
+        I.click(GWASUIAppProps.TeamProject1Select);
+        I.saveScreenshot('GwasTeamSelect1.png');
+        I.click(GWASUIAppProps.TeamProjectSubmitButton);
+        I.seeElement(GWASUIAppProps.TeamHeader);
+        const text = await I.grabTextFrom(GWASUIAppProps.TeamHeader);
+        expect(text).to.equal('Team Project / /gwas_projects/project1');
+    },
+
+    async changeTeamProject() {
+        I.click(GWASUIAppProps.ChangeTeamProject);
+        I.seeElement(GWASUIAppProps.TeamProjectSelectorBox);
+        I.click(GWASUIAppProps.TeamProjectSelectorDropdown);
+        I.click(GWASUIAppProps.TeamProject2Select);
+        I.saveScreenshot('GwasTeamSelect2.png');
+        I.click(GWASUIAppProps.TeamProjectSubmitButton);
+        I.seeElement(GWASUIAppProps.TeamHeader);
+        const text = await I.grabTextFrom(GWASUIAppProps.TeamHeader);
+        expect(text).to.equal('Team Project / /gwas_projects/project2');
+    },
```

Comment:
looks like this method is not used. Maybe remove it from this PR?

Reply:
I have added this  method for future use. This might save us time by just calling it in the main test.